src/test/java/org/reaktivity/nukleus/tcp/internal/streams/rfc793/ServerIOExceptionFromWriteIT.java

```diff
@@ -62,6 +63,7 @@
     @Rule
     public final TestRule chain = outerRule(SocketChannelHelper.RULE).around(reaktor).around(k3po).around(timeout);
 
+    @Ignore ("Should it work?")
```

Comment:
Yes it should!

Reply:
Actually it was a mistake in my test, now it's working.